### PR TITLE
Example for study and suite

### PR DESCRIPTION
**Pull request by mfeurer**
This pull request adds three new examples:

1. a simple example on suites
2. an advanced example on suites
3. an advanced example on studies

### CODE REVIEW

File: examples/30_extended/suites_tutorial.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+# We'll take a random subset of at least ten tasks of all available tasks on
+# the test server:
+all_tasks = list(openml.tasks.list_tasks().keys())
+num_tasks_in_suite = np.random.randint(10, len(all_tasks))
+task_ids_for_suite = sorted(np.random.choice(all_tasks, replace=False, size=num_tasks_in_suite))
+
+# The study needs a machine-readable and unique alias. To obtain this,
+# we simply append all task IDs to each other and hash them.
+alias = '-'.join([str(task_id) for task_id in task_ids_for_suite])
```
Comment (PGijsbers) [inline examples/30_extended/suites_tutorial.py]: again I think `uuid` is nicer to keep unrelated things brief.

File: examples/20_basic/simple_suites_tutorial.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+tasks = suite.tasks
+print(tasks)
+
+####################################################################################################
+# and iterated for benchmarking. For speed reasons we'll only iterate over the first three tasks:
+
+for task_id in tasks[:3]:
+    task = openml.tasks.get_task(task_id)
+    print(task)
+
```
[the remark this answers is not in the document]
Comment (mfeurer) [reply examples/20_basic/simple_suites_tutorial.py]: The reason I opted for this layout is that the print output of the two cells is separated.

File: examples/30_extended/study_tutorial.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+# Uploading studies
+# =================
+#
+# Creating a study is as simple as creating any kind of other OpenML entity.
+# In this examples we'll create a few runs for the OpenML-100 benchmark
+# suite which is available on the OpenML test server.
+
+openml.config.start_using_configuration_for_example()
+
+# Very simple classifier which ignores categorical features
```
[the remark this answers is not in the document]
Comment (mfeurer) [reply examples/30_extended/study_tutorial.py]: Good catch, it ignores that they are categorical, not that they exist.

File: examples/20_basic/simple_suites_tutorial.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (PGijsbers) [reply examples/20_basic/simple_suites_tutorial.py]: Makes sense, feel free to keep it this way

**Comment by PGijsbers** [comment]
CI: https://travis-ci.org/openml/openml-python/builds/597601289

File: examples/30_extended/study_tutorial.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+# Downloading studies
+# ===================
+
+############################################################################
+# This is done based on the dataset ID.
+study = openml.study.get_study(123)
+print(study)
+
+############################################################################
+# Suites also features a description:
```
Comment (ArlindKadra) [inline examples/30_extended/study_tutorial.py]: Study ?

File: examples/30_extended/study_tutorial.py [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+print(study.description)
+
+############################################################################
+# Suites are a container for runs:
```
Comment (ArlindKadra) [inline examples/30_extended/study_tutorial.py]: Same

**Pull request merged**